packages/react/src/Button/Button.examples.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import type {Meta} from '@storybook/react'
+import {Button} from '.'
+import {DownloadIcon} from '@primer/octicons-react'
+import {VisuallyHidden} from '../internal/components/VisuallyHidden'
+
+const meta: Meta<typeof Button> = {
+  title: 'Components/Button/Examples',
+} as Meta<typeof Button>
+
+export default meta
+
+export const LoadingStatusAnnouncementSuccessful = () => {
+  const [loading, setLoading] = React.useState(false)
+  const [success, setSuccess] = React.useState(false)
+
+  const resolveAction = async () => {
+    setLoading(true)
+    await new Promise(resolve => setTimeout(resolve, 1500))
+    setLoading(false)
+
+    return await true
+  }
+
+  const onClick = (resolveType: 'error' | 'success') => async () => {
+    const actionResult = await resolveAction()
+
+    if (resolveType === 'error') {
+      setSuccess(!actionResult)
+      return
+    }
+
+    setSuccess(actionResult)
+  }
+
+  return (
+    <>
+      <VisuallyHidden aria-live="polite">{!loading && success ? 'Export completed' : null}</VisuallyHidden>
+      <Button loading={loading} leadingVisual={DownloadIcon} onClick={onClick('error')}>
```

Comment:
```suggestion
      <Button loading={loading} leadingVisual={DownloadIcon} onClick={onClick('success')}>
```
Should this be `success`? I noticed that the live region doesn't get updated with the `Export completed` announcement in [the storybook success example](https://primer-ef8e619bcb-13348165.drafts.github.io/storybook/iframe.html?args=&id=components-button-examples--loading-status-announcement-successful&viewMode=story). 

It seems to be getting updated for the error example though!